test/index.spec.js

```diff
@@ -31,24 +31,24 @@ test('a default project should pass lint', async () => {
 
 test('should lint .ts file', async () => {
   const { stdout } = await lintProject('ts')
-  expect(stdout).toMatch('@typescript-eslint/no-empty-interface')
+  expect(stdout).toMatch('@typescript-eslint/no-unused-vars')
```

Comment:
This change is NOT correct.
This rule should NOT be firing, the other one should be.
This indicates that something in the stack is incorrectly integrating with typescript-eslint

Reply:
I think the reason of this is that `@typescript-eslint/no-empty-interface` is recommended rule for v5 but it's not for v6.
I think previously `stdout` has both errors `@typescript-eslint/no-empty-interface` and `@typescript-eslint/no-unused-vars` in v5 but now only `@typescript-eslint/no-unused-vars` due to the reason.

Reply:
@baseballyama @bradzacher any updates on this one? If I see correctly, this is the only thread that is blocking the MR from being merged.

Reply:
I believe this is fine.

Reply:
This is expected.
These tests aim to ensure the rules from `@typescript-eslint/eslint-plugin` are correctly applied to the files. Now that the `plugin:@typescript-eslint/recommended` config changes, the output should change accordingly. (`@typescript-eslint/no-empty-interface` removed; `@typescript-eslint/no-unused-vars` changed from `warn` to `error`)